client/components/deposits-overview/recent-deposits-list.tsx

```diff
@@ -133,8 +139,12 @@ const RecentDepositsList: React.FC< RecentDepositsProps > = ( {
 						{ strings.tableHeaders.amount }
 					</FlexItem>
 				</Flex>
+			</CardBody>
+			<CardDivider />
+			<CardBody className={ `${ tableClass }__container` }>
+				{ ' ' }
```

Comment:
I also wanted to confirm that this is intentional?

Reply:
That is not intentional, I thought that was part of the merge! Investigating.

Reply:
Removed, bad berge 🫠 f41d7b27f7f8ce7ec7831933d869639075160f85